modules/snooker/src/main/ts/ephox/snooker/resize/Bar.ts

```diff
@@ -10,7 +10,7 @@ const col = (column: number, x: number, y: number, w: number, h: number): SugarE
     width: w + 'px'
   });
 
-  Attribute.setAll(bar, { 'data-column': column, 'role': 'presentation' });
+  Attribute.setAll(bar, { 'data-mce-bogus': 'all', 'data-column': column, 'role': 'presentation' });
```

Comment:
Could we add a test for this so that we know that these bars doesn't get rendered as the `getContent()` or generate a undo level. Like fake hover the table, then call `editor.getContent()` and/or `undoManager.add()`